Readme.md

```diff
@@ -1,1266 +1,32 @@
-stream<gmail:MailThread,error?>|error threadList = gmailClient->listThreads(filter = {includeSpamTrash: false, 
-    labelIds: ["INBOX"]});
-
-if (threadList is stream<gmail:MailThread,error?>) {  
-    error? e = threadList.forEach(function (gmail:MailThread thread) {
-        log:printInfo(thread.toString());
-    }); 
-} else {
-    log:printError("Failed to list threads");
-}
-```
-Notes: Optionally we can provide `MsgSearchFilter` to specifically give optional parameters for this operation including
-**Whether to include SPAM or TRASH**, **maximum item count**, **specific label IDs**, **Page token** and **query**.
+### Setting up the prerequisites
 
-Sample is available at: https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail/blob/master/samples/threads/list_threads.bal
+1. Download and install Java SE Development Kit (JDK) version 11. You can install either [OpenJDK](https://adoptopenjdk.net/) or [Oracle JDK](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html).
 
-### Trash or un-trash threads
-This sample shows how to trash or un-trash an email thread that is available in the authorized user's account. When you 
-trash an email thread, it stays in your Trash folder for 30 days. After that time, it will be permanently deleted. So if 
-you want to recover the email thread you have deleted within these 30 days, you can do so using the un-trash capability 
-of the connector. The operation has These two operations returns `MailThread` record if the operations are successful. Else 
-returns `error`.
-
-```ballerina
-
-// ID of the thread to trash or un-trash.
-string sentMessageThreadId = "<THREAD_ID";
-
-log:printInfo("Trash thread");
-gmail:MailThread|error trash = gmailClient->trashThread(sentMessageThreadId);
-
-if (trash is gmail:MailThread) {
-    log:printInfo("Successfully trashed the thread");
-} else {
-    log:printError("Failed to trash the thread");
-} 
-
-log:printInfo("Un-trash thread");
-gmail:MailThread|error untrash = gmailClient->untrashThread(sentMessageThreadId);
-
-if (untrash is gmail:MailThread) {
-    log:printInfo("Successfully un-trashed the thread");
-} else {
-    log:printError("Failed to un-trash the thread");
-} 
-```
-Sample is available at: https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail/blob/master/samples/threads/trash_untrash_thread.bal
-
-### Delete thread
-This sample shows how to delete an email thread that is available in the authorized user's account.In contrast to 
-trashing a an email thread, delete thread permanently deletes the email thread such that it cannot be recovered again. 
-This operation returns nothing if successful. Else returns `error`.
-
-```ballerina
-
-// ID of the thread to delete.
-string sentMessageThreadId = "<THREAD_ID"; 
-
-error? delete = gmailClient->deleteThread(sentMessageThreadId);
+      > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed
```

Comment:
let's use markdown syntax, if this cannot be added as a step.

Reply:
We have agreed to have this as a block quotes in the guideline doc. https://docs.google.com/document/d/1WZg1ix9MpKDTQ_OOm3NVqhQ5L0VGNaUQ1LRmjp-SMO4/edit#heading=h.sk45j8pwb8h3